src/api/app/views/webui2/webui/groups/_group_members.html.haml

```diff
@@ -17,19 +17,20 @@
       - group.users.each do |user|
         .card.m-1.p-2.group-user
           .row.no-gutters
```

Comment:
This row will be empty if a user doesn't have write access, so rendering it isn't needed in this case.

Reply:
Sure! I'll move the condition out so the row won't be there if no icon is shown.

Anyway, the solution is not ideal, but this is the explanation:

The problem here is the position of the (X) icon to delete the member. 

Before, the icon was inside the second column and it didn't stay in the top right corner in small screens. And now that the box is fitting better for smaller screens it looks even uglier. Example:

![Screenshot-2019-8-21 Open Build Service](https://user-images.githubusercontent.com/2581944/63425992-1575ae00-c412-11e9-9a7d-af36be9332d2.png)

My first approach was putting the icon nomatter-where inside `.group-user` and playing with absolute position to fix it to the top right corner. However, elements with absolute position are not detected by Capybara.

Options I had:
a) Find some workaround to make the element clickable for Capybara.
b) Set the icon in its own row even if it takes its space on the top.

I choosed b). Do you have a better idea @dmarcoux?

![Screenshot-2019-8-21 Open Build Service(2)](https://user-images.githubusercontent.com/2581944/63427154-c54c1b00-c414-11e9-91b1-1f16b35e405e.png)